### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by harusoo** [skywind3000/atom-shell-commands#30]
I'm not sure if i'm the only one who's having this issue. But, I have been trying hours and hours to figure this out but it just doesn't work for me. I have no idea what's wrong.

**Comment by davidea72** [comment]
i don't know if we have the same problem, but after installing shell command, and add this contents to the file config.cson

 "atom-shell-commands":
    commands: [
      {
        name: "ls"
        command: "ls"
        arguments: [
          "{FileName}"
          "-o"
          "{FileNameNoExt}.exe"
        ]
        options:
          cwd: "{FileDir}"
          keymap: 'ctrl-2'
      }
    ]

nothing appear in the menu, only "stop command, go to first error, go to last error , go to next error , go to prev error"

i run atom 1.28.2 x64 on debian 9 and my config.cson is located in my home directory .atom/

**Comment by SleepyTonic** [comment]
I have the same issue. It appears the 'commands' key in config.cson is used by Atom. I got around this by changing the key to something else like 'commandz' and updating lines 44 and 46 in atom-shell-commands.js

**Comment by davidea72** [comment]
do you mean :
                        registeredAtomCommands = [];

                        if (value != null && value != undefined && 
                                value.commandz != null && value.commandz != undefined) {
                                // Register new commands
                                value.commandz.forEach( function( command ) {

                                        // Create an atom command for each entry
 
and in config.cson

"atom-shell-commands":
commands: [
{


if yes, it doesn't work for me

**Comment by SleepyTonic** [comment]
Yes in atom-shell-commands.js but in config.cson I changed it to...

> "atom-shell-commands":
>    commandz: [
>       { ...
>       
Also config.cson requires proper indentation.

**Comment by davidea72** [comment]
you are right!!! in the cut & copy i add some more space .... removed the space , now works
thanks

**Comment by Abdillah** [comment]
Does it means to customize the plugin to work? I experienced this issue too.

I think each plugin has `plugin.commands` config that define palette command.
Does this is a new enforcement? The plugin itself define the config item in `lib/config.js`. 
How strange!

```
MacOS High Sierra 10.13.5
Atom 1.34.0
Electron 2.0.16
Chrome 61.0.3163.100
Node v8.9.3
```
